engines/director/channel.cpp

```diff
@@ -366,9 +366,9 @@ void Channel::setCast(CastMemberID memberID) {
 	_height = _sprite->_height;
 	replaceWidget();
 
-	if (!_sprite->_puppet) {
+	if (!_sprite->_puppet && g_director->getVersion() >= 600) {
 		// Based on Director in a Nutshell, page 15
-		_sprite->_autoPuppet = true;
+		_sprite->setAutoPuppet(AutoPuppetProperty::kPuppetCast, true);
```

Comment:
In my opinion, it is way too verbose. I'd name it `kAPCast` or `kAutoPuppetCast`